frontend/appflowy_flutter/lib/plugins/database_view/board/application/board_bloc.dart

```diff
@@ -208,8 +224,22 @@ class BoardBloc extends Bloc<BoardEvent, BoardState> {
       },
     );
     final onGroupChanged = GroupCallbacks(
+      onGroupConfigurationChanged: (configurations) {
+        if (isClosed) return;
+        final config = configurations.first;
+        if (config.hideUngrouped) {
```

Comment:
Just a thought, maybe we should rename `hideUngrouped` and similar to `hideDefaultGroup` etc.